### ISSUE TITLE
Ringcentral widget is just ringing non stop when I am on call and other calls are coming at the same time

### ISSUE THREAD

**Issue by jinalptl1993** [ringcentral/ringcentral-embeddable#650]
I have integrated the RingCentral widget in my react js application. All working fine till now.

I have experienced one weird issue with it. When I was on a call, I had at least 3 other calls coming in and all ringing in non-stop while talking to an ongoing call. Is there any way to prevent this from happening? 

I think when I am using the ring central app it would just beep when another call came in if you were already on a call.

Thanks a lot in advance!

**Comment by embbnux** [comment]
Hi @jinalptl1993 Do you mean you don't have any way to ignore the incoming call? You can go to all calls list, click call item to ignore that to stop ringing. 

<img width="298" alt="image" src="https://user-images.githubusercontent.com/7036536/191895335-f175066b-839d-40c4-b665-b01fa86c5970.png">

**Comment by jinalptl1993** [comment]
@embbnux 
Thanks a lot for your quick response.

I mean to say when my call is running I keep hearing the ringing calls in the background.
I am able to see the screen as you displayed above but I need to go there and cut the existing call.

But my use case is I want to keep my current call going on and do nothing with other incoming calls.

**Comment by embbnux** [comment]
Hi @jinalptl1993 so you want to make the widget beep when you have call and have new incoming call ?

**Comment by jinalptl1993** [comment]
yeah, that's what I exactly want or somehow I don't want my existing call to get disturbed due to incoming.

**Comment by embbnux** [comment]
Thanks for the suggestion. It is a good point, I will talk with PM and add it into backlog

**Comment by jinalptl1993** [comment]
ok, Thanks a lot for all your help.

**Comment by embbnux** [comment]
supported with https://github.com/ringcentral/ringcentral-embeddable/pull/656
